test/Hast.Transformer.Vhdl.Tests/VerificationTests/XilinxSamplesVerificationTests.cs

```diff
@@ -10,8 +10,8 @@ public class XilinxSamplesVerificationTests : SamplesVerificationTestsBase
 {
     private static readonly string[] _devicesToTest =
     {
-        Nexys4DdrDriver.DeviceName,
-        AlveoU50Driver.DeviceName,
+        new Nexys4DdrDriver(timingReportParser: null).DeviceName,
+        new AlveoU50Driver(timingReportParser: null).DeviceName,
```

Comment:
Oublic consts can be used here too (`AlveoU50Driver` doesn't have one anymore but should).